src/viam/sdk/components/private/board_server.hpp

```diff
@@ -74,6 +74,11 @@ class BoardServer : public ResourceServer,
         const ::viam::component::board::v1::GetDigitalInterruptValueRequest* request,
         ::viam::component::board::v1::GetDigitalInterruptValueResponse* response) override;
 
+    ::grpc::Status StreamTicks(
+        ::grpc::ServerContext* context,
+        const ::viam::component::board::v1::StreamTicksRequest* request,
+        ::grpc::ServerWriter<::viam::component::board::v1::StreamTicksResponse>* writer) override;
+
```

Comment:
Assuming we're able to act on @acmorrow's suggestion about using `make_service_helper`, this should be marked `noexcept`. If that assumption is false then disregard :)